tensorflow/python/keras/callbacks.py

```diff
@@ -1305,7 +1305,10 @@ def on_epoch_end(self, epoch, logs=None):
     send = {}
     send['epoch'] = epoch
     for k, v in logs.items():
-      send[k] = v
+      if isinstance(v, (np.ndarray, np.generic)):
```

Comment:
Can you add a comment why these two types are special cases?

Reply:
Done